app/src/main/java/com/codingblocks/cbonlineapp/adapters/LeaderboardListAdapter.kt

```diff
@@ -9,10 +9,10 @@ import android.widget.TextView
 import com.codingblocks.cbonlineapp.R
 import com.codingblocks.onlineapi.models.Leaderboard
 import org.jetbrains.anko.AnkoLogger
-import java.util.*
 
-class LeaderboardListAdapter(internal var context: Context,
-                             var list: ArrayList<Leaderboard>?
+class LeaderboardListAdapter(
```

Comment:
remove changes in this class as it will fixed in #73

Reply:
This will fail in spotless check. Should I continue?

Reply:
@aggarwalpulkit596 this is the reason for the travis issue